src/TestSuite/TestCase.php

```diff
@@ -258,9 +257,6 @@ protected function tearDown(): void
         $this->getTableLocator()->clear();
         $this->_configure = [];
         $this->_tableLocator = null;
-        if (class_exists(Mockery::class)) {
-            Mockery::close();
```

Comment:
No longer needed, because `MockeryPHPUnitIntegration` trait takes care of it now